src/platforms/php/guides/symfony/index.mdx

```diff
@@ -48,6 +48,12 @@ SENTRY_DSN="___PUBLIC_DSN___"
 ###< sentry/sentry-symfony ###
 ```
 
+<Note>
+
+Performance monitoring integrations related to database, caching, and template rendering are disabled by default. To enable them, please see the [Integrations documentation](/platforms/php/guides/symfony/performance/pm-integrations/).
```

Comment:
This is not true: we've enabled it by default.

@HazAT if that is not desired let us know.

Reply:
Fixed.